### ISSUE TITLE
ng-module export

### ISSUE THREAD

**Issue by cgatian** [BeastCode/VSCode-Angular-TypeScript-Snippets#31]
Currently `ng-module` creates the following:
```
@NgModule({
  declarations: [],
  imports: [ CommonModule ],
  exports: [ FeatureModule ],
  providers: [],
  bootstrap: []
})
export class FeatureModule {}
```

I don't think it's necessary for the module to export itself.

**Comment by cgatian** [comment]
@BeastCode Im going to make a PR for this. It actually causes an error in lazy loaded modules

**Comment by BeastCode** [comment]
Thanks!!! It's been published: v4.2.15
